UM/TaskManagement/HttpRequestManager.py

```diff
@@ -193,6 +193,14 @@ def abortRequest(self, request: "HttpRequestData") -> None:
                     request.reply.abort()
                     Logger.log("d", "%s aborted", request)
 
+    @staticmethod
+    def readJSON(reply: QNetworkReply) -> Any:
+        """ Read a Json response into a Python object (list, dict, str depending on json type)"""
+        try:
+            return json.loads(bytes(reply.readAll()).decode("utf-8"))
+        except json.decoder.JSONDecodeError:
+            Logger.log("w", "Received invalid JSON for user subscribed packages from the Web Marketplace")
```

Comment:
Is this only to be used by the Marketplace?

Reply:
Must be a copy-paste error. Thanks.